submissions/servers/servers.md

```diff
@@ -0,0 +1,44 @@
+On the contrary, [Overwatch](https://playoverwatch.com) is a game that keeps track of most of things. Please excuse my non-knowledge of Overwatch gameplay, I just watched a video of the explanation and saw a bit of gameplay. The basis of Overwatch is pretty simple: it's a third-person-shooter game where there is a lot of ranged weapons. Overwatch largely pre-simulates the game too, such as when you fire a rocket, it's not waiting for the response of the server. Your client instead just simulates the rocket firing on your end, waiting for any change in the movement of the rocket from the server. If there is a change, your client corrects it automatically, which can cause a "snapping" effect - also observed with high latency and moving, your character snaps into place after a bit of movement. In Overwatch, this relates to rockets being fired. If you fire a rocket, your client only has a **prediction** of where and when the rocket is going to impact, so your client is working using that assumption. If the Overwatch server says that the rocket is somewhere else, your client changes the position of the rocket to what the server says it is. When you hit someone with the rocket and give them damage, the damage is **not given until the server acknowledges the packets of the rocket being sent**, which means that the damage animation doesn't render until the server responds. The difference of these two implementations of the Lockstep algorithm are that Overwatch monitors every item, player, bullet etc. and only corrects your client if something went wrong, whilst Factorio monitors only player inputs and throws a desync if something goes wrong (I'll explain what a desync is later). How Overwatch works has all been explained [here (shorter video)](https://www.youtube.com/watch?v=vTH2ZPgYujQ) and [maybe here too](https://www.youtube.com/watch?v=W3aieHjyNvw) by Overwatch developers in higher detail with additional information about Overwatch's network handling. 
+The reason for which Factorio uses it's version of the lockstep algorithm is that Factorio has a lot more inputs than Overwatch; there are logistic bots, miners, trains, spidertrons, biters, players and all of that adds up to a large amount of data that would need to be transferred every tick. Instead, it is much easier to just give the client the map when they connect and tell them any other inputs that would alter the simulation, such as a player moving 10 tiles to the right etc. See image below.
```

Comment:
I think here you need to put focus on that Factorio will have to essentially send the entire save every tick. Not just bots, trains, etc, but every belt, assembler, light, power-pole, tree and where every inserter's hand is. It's a tonnnn of data that will be required.